src/utils/basic.ts

```diff
@@ -47,12 +47,16 @@ export const getGroupLink = (url): string => {
  * @returns
  */
 export const getByPath = (obj, path) => {
-  path.split(".").forEach(function (level) {
-    if (!obj) {
-      return;
-    }
-    obj = obj[level];
-  });
+  path
+    .replace(/\[/g, ".")
+    .replace(/]/g, "")
```

Comment:
Before my refactoring there was some logic written for bracket in real world you should just use dots as seprator as you really don't need brackets when using getById